cmd/check_test.go

```diff
@@ -104,14 +104,46 @@ func TestCheckSlack(t *testing.T) {
 
 	createEmptyGitRepo(t)
 
-	slack := 5
+	slack := 5.0
+	usePercents:=false
 
-	runCheckPS(t, "pageweight", slack, true, "gzippedjs,10")
-	runCheckPS(t, "pageweight", slack, false, "gzippedjs,15")
+	runCheckPS(t, "pageweight", slack, usePercents, true, "gzippedjs,10")
+	runCheckPS(t, "pageweight", slack, usePercents, false, "gzippedjs,15")
 
 	t.Logf("Running check command p: %s w: %t i: %s", "pageweight", false, "gzippedjs,16")
 
-	errCode := Check("pageweight", slack, false, "csv", false, strings.NewReader("gzippedjs,16"))
+	errCode := Check("pageweight", slack, usePercents, false, "csv", false, strings.NewReader("gzippedjs,16"))
+
+	if errCode != 50 {
+		t.Fatalf("Check command passed unexpectedly!")
+	}
+}
+
+func TestCheckSlackPercent(t *testing.T) {
+	if testing.Verbose() {
+		log.SetLogThreshold(log.LevelInfo)
+		log.SetStdoutThreshold(log.LevelInfo)
+	}
+
+	createEmptyGitRepo(t)
+
+	slack := 20.0
+	usePercents:=true
+
+	runCheckPS(t, "pageweight", slack, usePercents, true, "gzippedjs,100")
+	runCheckPS(t, "pageweight", slack, usePercents, false, "gzippedjs,101")
+
+	t.Logf("Running check command p: %s w: %t i: %s", "pageweight", false, "gzippedjs,120")
+
+	errCode := Check("pageweight", slack, usePercents, false, "csv", false, strings.NewReader("gzippedjs,120"))
+
+	if errCode != 0 {
+		t.Fatalf("Check command failed unexpectedly!")
+	}
+
+	t.Logf("Running check command p: %s w: %t i: %s", "pageweight", false, "gzippedjs,160")
```

Comment:
Can we tighten the higher value so it's just above the threshold - so 121 instead of 160?